scripts/enable_sonar.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -ex
+
+echo "Fetching repo id"
+REPO_ID=$(curl --fail -sL -H "Authorization: token ${GITHUB_TOKEN}" https://api.github.com/repos/${GITHUB_REPOSITORY} | jq --raw-output ".id")
+# GITHUB_REPOSITORY name will be org_name/repo_name
+
+if [ -z "$REPO_ID" ]; then
+    echo "Failed: Repo id is empty"
+    exit 1
+else
+    echo "Printing repo id : $REPO_ID"
+fi
+
+echo "Running provision_projects"
+curl --fail --include \
+    --request POST \
+    --header "Content-Type: application/x-www-form-urlencoded" \
+    -u ${SONAR_TOKEN}: \
+    --data-binary "installationKeys=${GITHUB_REPOSITORY}%7C${REPO_ID}&organization=${SONAR_ORG}" \
+    "https://sonarcloud.io/api/alm_integration/provision_projects"
+
+SONAR_PROJECT_KEY=$(echo ${GITHUB_REPOSITORY} | sed 's/\//_/')
```

Comment:
Where do you get this from https://github.com/variant-inc/actions-dotnet/blob/f6c038b536e1895d4ae03990a57220c7a4b3fe92/scripts/sonarscan.sh#L16?

Reply:
We get sonar project key from here https://github.com/variant-inc/actions-setup/blob/8a0f48d7e99ea62109d16a35440980af9c2f2370/action.yml#L20. Basically i am doing same thing here while creating sonar project.

Reply:
Is this removed?